ant-node/src/networking/driver/mod.rs

```diff
@@ -175,6 +180,7 @@ impl SwarmDriver {
         let mut relay_manager_reservation_interval = interval(RELAY_MANAGER_RESERVATION_INTERVAL);
         let mut bootstrap_interval = Some(interval(BOOTSTRAP_CHECK_INTERVAL));
         let mut dial_queue_check_interval = interval(DIAL_QUEUE_CHECK_INTERVAL);
+        let mut network_wide_replication_interval = interval(NETWORK_WIDE_REPLICATION_INTERVAL);
```

Comment:
better scrammable the interval to avoid nodes' synced actions causing spike on resource usage